src/applications/caregivers/tests/unit/components/ApplicationDownloadLink.unit.spec.jsx

```diff
@@ -38,106 +44,136 @@ describe('CG <ApplicationDownloadLink>', () => {
     localStorage.setItem('csrfToken', 'my-token');
     apiRequestStub = sinon.stub(api, 'apiRequest');
     recordEventStub = sinon.stub(recordEventModule, 'default');
-    sinon.stub(URL, 'createObjectURL').returns('my_stubbed_url.com');
-    sinon.stub(URL, 'revokeObjectURL');
   });
 
   afterEach(() => {
     localStorage.clear();
-    sinon.restore();
-  });
-
-  context('default behavior', () => {
-    it('should render a download file button', () => {
-      const { selectors } = subject();
-      const { vaAlert, vaLink, vaLoadingIndicator } = selectors();
-      expect(vaLink).to.exist;
-      expect(vaAlert).to.not.exist;
-      expect(vaLoadingIndicator).to.not.exist;
-    });
+    apiRequestStub.restore();
```

Comment:
You can keep this as `sinon.restore();` It should clean up all stubs, regardless of where they are declared.